src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+  const declarations = sourceString.split(';');
+
+  for (const declaration of declarations) {
+    const parts = declaration.split(':');
+
+    if (parts.length === 2) {
+      const property = parts[0].trim();
+
+      stylesObject[property] = parts[1].trim();
+    }
+  }
```

Comment:
According to the checklist, loops should be avoided in this task. Instead, use iteration methods like `map`, `filter`, or `reduce` to transform the string into an object. For example, you can split the string by semicolons, filter out any empty entries, and then use `reduce` to accumulate the entries into an object.